main/src/cgeo/geocaching/filters/core/GeocacheFilterType.java

```diff
@@ -18,6 +18,7 @@
     PERSONAL_NOTE("note", R.string.cache_filter_personalnote, R.string.cache_filtergroup_userspecific, PersonalNoteGeocacheFilter::new),
     DIFFICULTY("difficulty", R.string.cache_filter_difficulty, R.string.cache_filtergroup_details,  DifficultyGeocacheFilter::new),
     TERRAIN("terrain", R.string.cache_filter_terrain, R.string.cache_filtergroup_details, TerrainGeocacheFilter::new),
```

Comment:
I think that TERRAIN and DIFFICULTY should be removed from the type list

Reply:
I think I need the types for creating the views.

Reply:
Removed from selection list with commit 90a149e